specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json

```diff
@@ -1924,7 +1924,7 @@
         },
         "status": {
           "enum": [
-            "notstarted",
+            "notStarted",
             "running",
             "succeeded",
             "failed",
```

Comment:
Should `partiallycompleted` be also changes to `partiallyCompeted` ?

Reply:
@suhas92 I do not know, you tell me :)

Reply:
We should loop in @tasharm-0412, he may be able to help answer this question.  I'm looking at the code now but can't find where these values are serialized into JSON so any help in determining what all of the enum values should be would be much appreciated.

Reply:
Ok, actually I think I found it.  Here is how the job status enum is specified on the service side:

```
public enum JobStatus
    {
        Unknown = 0,
        NotStarted,
        Running,
        Cancelling,
        Cancelled,
        PartiallySucceeded,
        Succeeded,
        Failed
    }
```

So my guess is that the serialized versions of all of these values will be camel case, so it is reasonable to assume that "NotStarted" -> "notStarted" and "PartiallySucceeded" -> "partiallySucceeded".

Reply:
Thanks I made the change. How about the analysis task state? is it came case too?

Reply:
Let me see if I can figure out how the orchestrator is representing this.  From the TA side we will be using the same enum, but I need to find out whether or not these values are passed through the orchestrator or if they do any manipulation of the response.

Reply:
I see this is already merged, but just to follow up it looks like the enum used on the orchestrator side matches what we have in TA, so this should be fine for both services.